02-microarray/pathway-analysis_microarray_03_gsea.Rmd

```diff
@@ -404,30 +414,35 @@ Let's look for the gene set with the highest positive NES.
 
 ```{r}
 gsea_result_df %>%
-  # Combining the `dplyr::desc()` function with `dplyr::arrange()` sorts the
-  # values in descending order
-  dplyr::arrange(dplyr::desc(NES))
+  # Use `top_n()` to return the top n observation using `NES` as the ordering variable
+  dplyr::top_n(n = 1, wt = NES)
```

Comment:
Can print out more than 1 just to show a little bit of a preview? Not 100% necessary but might be nice.